Dockerfile

```diff
@@ -1,40 +1,54 @@
 ##
 # osgeo/proj.4
 
-FROM ubuntu:18.04
+FROM ubuntu:18.04 as builder
 
 MAINTAINER Howard Butler <howard@hobu.co>
 
+ARG PROJ_VERSION=master
+ARG DESTDIR="/build"
 
 # Setup build env
-RUN mkdir /build
 RUN apt-get update -y \
     && apt-get install -y --fix-missing --no-install-recommends \
             software-properties-common build-essential ca-certificates \
             git make cmake wget unzip libtool automake \
-            zlib1g-dev \
-    && apt-get remove --purge -y $BUILD_PACKAGES  && rm -rf /var/lib/apt/lists/*
-
-
-RUN mkdir /vdatum \
-    && cd /vdatum \
-    && wget http://download.osgeo.org/proj/vdatum/usa_geoid2012.zip && unzip -j -u usa_geoid2012.zip -d /usr/share/proj \
-    && wget http://download.osgeo.org/proj/vdatum/usa_geoid2009.zip && unzip -j -u usa_geoid2009.zip -d /usr/share/proj \
-    && wget http://download.osgeo.org/proj/vdatum/usa_geoid2003.zip && unzip -j -u usa_geoid2003.zip -d /usr/share/proj \
-    && wget http://download.osgeo.org/proj/vdatum/usa_geoid1999.zip && unzip -j -u usa_geoid1999.zip -d /usr/share/proj \
-    && wget http://download.osgeo.org/proj/vdatum/vertcon/vertconc.gtx && mv vertconc.gtx /usr/share/proj \
-    && wget http://download.osgeo.org/proj/vdatum/vertcon/vertcone.gtx && mv vertcone.gtx /usr/share/proj \
-    && wget http://download.osgeo.org/proj/vdatum/vertcon/vertconw.gtx && mv vertconw.gtx /usr/share/proj \
-    && wget http://download.osgeo.org/proj/vdatum/egm96_15/egm96_15.gtx && mv egm96_15.gtx /usr/share/proj \
-    && wget http://download.osgeo.org/proj/vdatum/egm08_25/egm08_25.gtx && mv egm08_25.gtx /usr/share/proj \
-    && rm -rf /vdatum
-
+            zlib1g-dev libsqlite3-dev pkg-config sqlite3
 
 RUN git clone https://github.com/OSGeo/proj.4.git \
     && cd proj.4 \
     && ./autogen.sh \
     && ./configure --prefix=/usr \
     && make \
-    && make install \
-    && rm -rf /proj.4
+    && make install
+
+
+
+
+FROM ubuntu:18.04 as runner
+
+RUN date
+
+RUN apt-get update; \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+        libsqlite3-0 \
+        curl unzip
+
+COPY --from=builder  /build/usr/bin/ /usr/bin/
+COPY --from=builder  /build/usr/lib/ /usr/lib/
+COPY --from=builder  /build/usr/include/ /usr/include/
+COPY --from=builder  /build/usr/share/proj/ /usr/share/proj/
+
+RUN \
```

Comment:
you should use the proj-datumgrid-XXXXXX packages instead. Some of the below grids are deprecated

Reply:
will do.